File: Superyatzy/superyatzy.cpp
#include <algorithm>
#include <iostream>

int main() {
    std::ios::sync_with_stdio(false);
    std::cin.tie(nullptr);

    int N, M;
    std::cin >> N >> M;

    int counts[7] = {0};

    for (int i = 0; i < N; i++) {
        int t;
        std::cin >> t;
        
        counts[t]++;
    }

    int maxCount = *std::max_element(counts + 1, counts + 7);

    if (N - maxCount <= M) {
        std::cout << "Ja\n";
    } else {
        std::cout << "Nej\n";
    }

    return 0;
}

File: Superyatzy/superyatzy.py
N, M = map(int, input().split())

counts = [0] * 7

for i in range(N):
    t = int(input())
    
    counts[t] += 1

maxCount = 0

for i in range(1, 7):
    if counts[i] > maxCount:
        maxCount = counts[i]

if N - maxCount <= M:
    print("Ja")
else:
    print("Nej")
